src/App.css

```diff
@@ -0,0 +1,18 @@
+.wrapper {
+  width: 95%;
+  margin: 0 auto;
+}
+
+
+.nav {
+  height: 30px;
+  background-color: #fff;
+  margin-bottom: 30px;
+  border: 1px solid #ddd;
+}
+.nav h2{
```

Comment:
Add space after h2, I'd suggest you set up [prettier](https://glebbahmutov.com/blog/configure-prettier-in-vscode/) on your editor for easier code formatting